dev_guide/managing_images.adoc

```diff
@@ -116,6 +116,37 @@ endif::[]
 that prevents the registry client from pulling from such a tag.
 ====
 
+[[tag-naming]]
+=== Tag Naming
+Prefer more generic names for tags so that image pruning can get you rid of old
+images.
+
+Images evolve over the time and tag reflects it. It always points to the latest
```

Comment:
Here tag is ISTAG? I guess so, then I'd propose using istag, instead. Just tag might be confusing.